doc/api_ref/pubkey.rst

```diff
@@ -99,6 +99,15 @@ Kyber
 
 Post-quantum key encapsulation scheme based on lattices.
 
+.. note::
+
+   Currently two modes for Kyber are defined: the round3 specification
+   from the NIST PQC competetition, and the "90s mode" (which uses
```

Comment:
Lost in rebase or force push?

```suggestion
   from the NIST PQC competition, and the "90s mode" (which uses
```

Reply:
There were several spelling errors in this file, fixed in baea85c